### PR TITLE
Bugfix/contrition

### PR DESCRIPTION
**Pull request by oisinod**

### CODE REVIEW

File: src/analysis/retail/priest/discipline/CHANGELOG.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { SpellLink } from 'interface';
 
 
 export default [
+  change(date(2022, 10,15), <>Fixed <SpellLink id={TALENTS_PRIEST.CONTRITION_TALENT.id}/>.</>, Hana),
   change(date(2022, 10,11), <>Added <SpellLink id={TALENTS_PRIEST.CRYSTALLINE_REFLECTION_TALENT.id}/> module showing it's damage breakdown.</>, Hana),
   change(date (2022,10,10), <>Added generic <SpellLink id={SPELLS.POWER_WORD_SHIELD.id}/> module which handles attribution of it's amplifiers</>, Hana),
   change(date(2022, 10, 8), <>Added <SpellLink id={TALENTS_PRIEST.AEGIS_OF_WRATH_TALENT.id}/>.</>, Hana),
```
Comment (Pewtro) [inline src/analysis/retail/priest/discipline/CHANGELOG.tsx]: please fix the spacings here, it's making me cry :(

**Review by Pewtro — APPROVED** [review]
lgtm but the spacing in the changelog is a sin

File: src/analysis/retail/priest/discipline/CHANGELOG.tsx (hunk 1 above)
Comment (oisinod) [reply src/analysis/retail/priest/discipline/CHANGELOG.tsx]: @Pewtro done 👍

**Comment by kfinch** [comment]
Merge conflict in changelog - was there a talent rename?

**Comment by Pewtro** [comment]
> Merge conflict in changelog - was there a talent rename?

It's just from this PR being merged https://github.com/WoWAnalyzer/WoWAnalyzer/commit/97aac678d7bb282530050b63f04e8426ef312b54

**Pull request merged**